index.html

```diff
@@ -16,30 +18,33 @@
 <body>
     <nav  id="navber">
         <ul id="nave_menu">
+            
+            <img src="/Assets/college_logo.jpg.jpg" alt="loding image " class="logo_1">
```

Comment:
Use the following logo path under the src parameter: 

`<img src="/Assets/Favicon.png" alt="loding image " class="logo_1">`